scripts/setup/create_singlenode_container.sh

```diff
@@ -54,18 +53,18 @@ server_exec() {
     server_exec 'tmux new -d -s cluster'
     server_exec 'tmux send-keys -t containerd "sudo containerd" ENTER'
     sleep 3
-    server_exec 'pushd ~/vhive/scripts > /dev/null && ./setup_tool create_one_node_cluster stock-only && popd > /dev/null'
-    server_exec 'tmux send-keys -t cluster "watch -n 0.5 kubectl get pods -A" ENTER'
 
-    # Setup github authentication.
-    ACCESS_TOKEH="$(cat $GITHUB_TOKEN)"
+    server_exec 'pushd ~/vhive/scripts > /dev/null && ./setup_tool prepare_one_node_cluster stock-only && popd > /dev/null'
+    server_exec 'kubectl label nodes --all loader-nodetype=singlenode'
+    server_exec 'pushd ~/vhive/scripts > /dev/null && ./setup_tool setup_master_node stock-only && popd > /dev/null'
+
+    server_exec 'tmux send-keys -t cluster "watch -n 0.5 kubectl get pods -A" ENTER'
 
-    server_exec 'echo -en "\n\n" | ssh-keygen -t rsa'
-    server_exec 'ssh-keyscan -t rsa github.com >> ~/.ssh/known_hosts'
-    # server_exec 'RSA=$(cat ~/.ssh/id_rsa.pub)'
+    # Enable affinity
+    server_exec 'kubectl apply -f ~/loader/config/enable_affinity.yaml'
 
-    server_exec 'curl -H "Authorization: token '"$ACCESS_TOKEH"'" --data "{\"title\":\"'"key:\$(hostname)"'\",\"key\":\"'"\$(cat ~/.ssh/id_rsa.pub)"'\"}" https://api.github.com/user/keys'
-    # server_exec 'sleep 5'
+    # Patch init scale
+    server_exec 'cd loader; bash scripts/setup/patch_init_scale.sh'
 
     $DIR/expose_infra_metrics.sh $SERVER
```

Comment:
Make this conditional as it is done in setup_multinode (based on DPELOY_PROMETHEUS)